geomesa-accumulo/geomesa-accumulo-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/GeoMesaAccumuloInputFormat.scala

```diff
@@ -134,6 +133,22 @@ object GeoMesaAccumuloInputFormat extends LazyLogging {
    * @param plan query plan
    */
   def configure(conf: Configuration, params: java.util.Map[String, _], plan: AccumuloQueryPlan): Unit = {
+    val auths = AccumuloDataStoreParams.AuthsParam.lookupOpt(params).map(a => new Authorizations(a.split(","): _*))
+    configure(conf, params, plan, auths)
+  }
+
+  /**
+   * Configure the input format based on a query plan
+   *
+   * @param conf configuration to update
+   * @param params data store parameters
+   * @param plan query plan
+   */
+  def configure(
```

Comment:
What's the goal of having two different `configure` methods?  Is it so that one can query with more restricted Auths than they have available?

Reply:
no, it was just to keep the old one there for api compatibility.

Reply:
could maybe deprecate the old one, but i forget if it's actually still used anywhere.